wrappers/python/xraydb/xraydb.py

```diff
@@ -430,8 +430,29 @@ def xray_lines(self, element, initial_level=None, excitation_energy=None):
                                                r.initial_level, r.final_level)
         return out
 
+    def xray_line_strengths(self, element):
+        """
+        return the absolute line strength in cm^2/gr for all available lines
+
+        arguments
+        ---------
+        element:  atomic number, atomic symbol for element
+
+        """
+        out = {}
+        for label, eline in self.xray_lines(element).items():
+            try:
+                edge = self.xray_edge(element, eline.initial_level)
+                mu = self.mu_elam(element, [edge.absorption_edge-2,
+                                            edge.absorption_edge+2], kind='photo')
+                out[label] = (mu[1]-mu[0]) * eline.intensity * edge.fluorescence_yield
```

Comment:
It's   mu_edge * yield * intensity.   Isn't that the same as the xraylib paper, except that the CK probabilities appear to be already included in this data set....?   I guess the question is "do these results agree with the results from xraylib?"    If/when they don't agree, I'm not sure how to tell which one is right, or what the difference might be....